.prettierrc

```diff
@@ -1,4 +1,7 @@
 {
```

Comment:
This is stuff for Prettier, which is a code formatter thing. Just some rules for styling code, dosen't actually affect the functionality.